dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/ApplicationManager.java

```diff
@@ -137,6 +138,9 @@ protected void modified(ComponentContext compcontext, Map<String, Object> proper
         setUseJandex(useJandexValue == null ? false : useJandexValue);
         //System.setProperty("com.ibm.ws.jandex.enable", useJandexValue.toString()); // Temporary -- REMOVE THIS LATER ////
 
+        String annotationScanLibraryValue = getProperty(properties, "annotationScanLibrary", "");
+        setAnnotationScanLibrary(annotationScanLibraryValue);
```

Comment:
Also l believe this should be surrounded by a beta guard to not be executed unless you are using the beta.